test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/argument-string-non-iso-calendar.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
This doesn't appear to be different from `test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/argument-string-iso-calendar.js`, which makes sense, because you can't get a custom calendar from an ISO string. Let's just remove this.

Reply:
trashed